src/mock/fish-data.js

```diff
@@ -1,3 +1,153 @@
+import { getRandomInteger, times } from './utils.js';
+
 const GLOBAL_INTEGER = 3;
 
-export { GLOBAL_INTEGER };
+const DESCRIPTIONS = ['Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  'Cras aliquet varius magna, non porta ligula feugiat eget.',
+  'Fusce tristique felis at fermentum pharetra.', 'Aliquam id orci ut lectus varius viverra.',
+  'Nullam nunc ex, convallis sed finibus eget, sollicitudin eget ante.', 'Phasellus eros mauris, condimentum sed nibh vitae, sodales efficitur ipsum.',
+  'Sed blandit, eros vel aliquam faucibus, purus ex euismod diam, eu luctus nunc ante ut dui.',
+  'Sed sed nisi sed augue convallis suscipit in sed felis.',
+  'Aliquam erat volutpat. Nunc fermentum tortor ac porta dapibus.',
+  'In rutrum ac purus sit amet tempus.'
+];
+
+const PLACES = ['Dubay',
+  'Tirane',
+  'Chaco',
+  'Vienna',
+  'Lankaran',
+  'Flanders',
+  'Moscow',
+  'Paris',
+  'New-York'
+];
+
+const VEHICLE_TYPE = [
+  'taxi',
+  'bus',
+  'train',
+  'ship',
+  'drive',
+  'flight',
+  'check-in',
+  'sightseeing',
+  'restaurant'
+];
+
+
+/* destinations */
+
+const createDestination = () => ({
+  id: 1,
+  description: DESCRIPTIONS[getRandomInteger(0, DESCRIPTIONS.length - 1)],
+  name: PLACES[getRandomInteger(0, PLACES.length - 1)],
+  pictures: [
+    {
+      src: 'http://picsum.photos/300/200?r=0.0762563005163317',
+      description: DESCRIPTIONS[getRandomInteger(0, DESCRIPTIONS.length - 1)]
+    }
+  ]
+});
+
+const createDestinationIdPlus = () => {
+  const setDestinations = [];
+  for (let i = 0; i < GLOBAL_INTEGER; i++) {
+    const nextDestination = createDestination();
+    setDestinations.push(nextDestination);
+    nextDestination.id += i;
+  }
+  return setDestinations;
```

Comment:
```suggestion
  const setDestinations = [];
  
  for (let i = 0; i < GLOBAL_INTEGER; i++) {
    const nextDestination = createDestination();
    setDestinations.push(nextDestination);
    nextDestination.id += i;
  }
  
  return setDestinations;
```

обычно разные блоки кода разделяют пустыми строками